src/lib_c/aarch64-linux-gnu/c/sys/types.cr

```diff
@@ -7,7 +7,7 @@ lib LibC
   alias ClockT = Long
   alias ClockidT = Int
   alias DevT = ULong
-  alias GidT = UInt
+  alias GidT = UInt32T
```

Comment:
Minor: this is actually `unsigned int` in linux (all GNU/musl-libc headers) so `LibC::UInt` in Crystal. On BSD they resolve to `__uint32_t` which also resolves to `unsigned int`. In the end this resolves to the same Crystal type (`UInt32`), so we can spare these changes in every `sys/types.h` bindings.